spec/Generics.tex

```diff
@@ -868,6 +868,81 @@ \section{User-Defined Compiler Diagnostics}
 
 \end{chapelexample}
 
+\section{Declaring Specific Instantiations}
+\label{Declaring_Specific_Instantiations}
+\index{specific instantiations}
+\index{generic specialization}
+
+The Chapel language facility supports three mechanisms for creating generic
+functions along with concrete functions that work with specific types.
+One can use these mechanisms to create specialized versions of generic
+functions.
```

Comment:
... three mechanisms for *using* generic functions ...

I"d drop "that work with specific types".

"to create specialized versions" -- I think you mean "to choose between" generic and/or concrete functions. The last sentence above does not seem right to me as-is. Maybe blend it with the first sentence in the paragraph by using "choose between" alongside with "use" verbs somehow.

I suggest replacing, throughout this section, "specialization" with "instantiation" to be consistent with other parts of the spec.